datasource/commit/test-fixtures/template.pkr.hcl

```diff
@@ -23,6 +23,7 @@ build {
       "echo 'num_branches: ${length(data.git-commit.test.branches)}'",
       "echo 'author: ${data.git-commit.test.author}'",
       "echo 'committer: ${data.git-commit.test.committer}'",
+      "echo 'timestamp: ${data.git-commit.test.timestamp}'",
```

Comment:
Can you please also add the timestamp line to [the acceptance tests](https://github.com/ethanmdavidson/packer-plugin-git/blob/main/datasource/commit/data_acc_test.go)? You just need to create a regex to match the line with the timestamp, and then use it in the `checkMatch` function. See the [commit message regex](https://github.com/ethanmdavidson/packer-plugin-git/blob/main/datasource/commit/data_acc_test.go#L56) and [commit message check](https://github.com/ethanmdavidson/packer-plugin-git/blob/main/datasource/commit/data_acc_test.go#L64) as an example.

Reply:
thanks for the feedback. I just added the missing lines in https://github.com/ethanmdavidson/packer-plugin-git/pull/111/commits/94574c9af23ddbf5ba5ed62be7e4655046f9e09f